cloverage/src/cloverage/instrument.clj

```diff
@@ -1,18 +1,19 @@
 (ns cloverage.instrument
-  (:require [clojure.tools.logging :as log]
+  (:require [clojure.pprint :as pprint]
+            [clojure.tools.logging :as log]
             [clojure.tools.reader :as r]
             [cloverage.debug :as d]
             [cloverage.rewrite :refer [unchunk]]
             [cloverage.source :as s]
             [riddley.walk :as rw]
             [slingshot.slingshot :refer [throw+]]))
 
-(defn iobj? [form]
+(defn- iobj? [form]
```

Comment:
There are a bunch of functions used only in this namespace. I made them private so it's easier to refactor this namespace going forward -- this way we don't need to try to work out if they're used anywhere else.